src/Views/Config.elm

```diff
@@ -3,12 +3,12 @@ module Views.Config exposing (appTitle, email, helpline, helplineDisplay)
 
 appTitle : String
 appTitle =
-    "Pics, stories & info"
+    "Contemplating Action"
 
 
 email : String
 email =
-    "katja@neontribe.co.uk"
+    "ca@neontribe.co.uk"
```

Comment:
Did we agree this? Or going with contemplatingaction@neontribe.co.uk until we get a domain set up?

Reply:
Did we set up the group yet?

Reply:
ca@neontribe.co.uk is what is currently in Kat's copy

Reply:
:+1: OK let's go with it. We can change.